packages/trader/src/App/Containers/RealAccountSignup/add-crypto-currency.jsx

```diff
@@ -64,33 +68,50 @@ class AddCryptoCurrency extends Component {
                     isSubmitting,
                 }) => (
                     <form onSubmit={handleSubmit}>
-
-                        {!this.can_add_fiat && this.props.available_crypto_currencies.length !== 0 &&
+                        {!this.can_add_fiat &&
```

Comment:
:ok_hand:

Reply:
@reza-fs lol what is it? did i deleted something that shouldn't be deleted?